rules/msk_topic_config_test.go

```diff
@@ -84,13 +99,92 @@ resource "kafka_topic" "topic_with_incorrect_repl_factor" {
 				},
 			},
 		},
+		{
+			name: "missing config attribute",
+			input: `
+resource "kafka_topic" "topic_without_config" {
+  name = "topic_without_config"
+  replication_factor = 3
+}`,
+			expected: []*helper.Issue{
+				{
+					Rule:    rule,
+					Message: "missing config attribute: the topic configuration must be specified in a config attribute",
+					Range: hcl.Range{
+						Filename: fileName,
+						Start:    hcl.Pos{Line: 2, Column: 1},
+						End:      hcl.Pos{Line: 2, Column: 46},
+					},
+				},
+			},
+		},
+		{
+			name: "missing compression type",
+			input: `
+resource "kafka_topic" "topic_without_compression_type" {
+  name               = "topic_without_compression_type"
+  replication_factor = 3
+  config = {
+  }
+}`,
+			fixed: `
+resource "kafka_topic" "topic_without_compression_type" {
+  name               = "topic_without_compression_type"
+  replication_factor = 3
+  config = {
+    "compression.type" = "zstd"
+  }
+}`,
+			expected: []*helper.Issue{
+				{
+					Rule:    rule,
+					Message: "missing compression.type: it must be equal to 'zstd'",
+					Range: hcl.Range{
+						Filename: fileName,
+						Start:    hcl.Pos{Line: 5, Column: 3},
+						End:      hcl.Pos{Line: 6, Column: 4},
+					},
+				},
+			},
+		},
+		{
+			name: "wrong compression type",
```

Comment:
could we add a 'happy path' test, i.e. test that when you give the correct configuration there is no issue? To avoid potential regressions in the future

Reply:
good point